### PR TITLE
Add test cases that is left commented out

### PR DESCRIPTION
**Pull request by JonathanHoshi**
Update the test cases of ModelManagerTest.java, UserPrefsTest.java

### CODE REVIEW

**Review by arcornior — APPROVED** [review]
LGTM!

File: src/test/java/seedu/trackermon/model/ModelManagerTest.java [hunk 1]
```diff
@@ -1,132 +1,133 @@
...
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.trackermon.commons.core.GuiSettings;
+import seedu.trackermon.model.show.NameContainsKeywordsPredicate;
+import seedu.trackermon.testutil.ShowListBuilder;
+
+public class ModelManagerTest {
```
Comment (lowkaiwei98) [inline src/test/java/seedu/trackermon/model/ModelManagerTest.java:21]: Please add header comment?

File: src/test/java/seedu/trackermon/model/UserPrefsTest.java [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-//    }
-//
-//}
+package seedu.trackermon.model;
+
+import static seedu.trackermon.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class UserPrefsTest {
```
Comment (lowkaiwei98) [inline src/test/java/seedu/trackermon/model/UserPrefsTest.java:7]: Please add header comment?

File: src/test/java/seedu/trackermon/model/ModelManagerTest.java (hunk 1 above)
Comment (JonathanHoshi) [reply src/test/java/seedu/trackermon/model/ModelManagerTest.java:21]: A lot of the other test classes do not have a header comment, is someone going to update the all test cases to include a javadoc header?

Comment (lowkaiwei98) [reply src/test/java/seedu/trackermon/model/ModelManagerTest.java:24]: i opened a new issue for this, you can go there and discuss about who is going update what.

File: src/test/java/seedu/trackermon/model/UserPrefsTest.java (hunk 2 above)
Comment (lowkaiwei98) [reply src/test/java/seedu/trackermon/model/UserPrefsTest.java:10]: i opened a new issue for this, you can go there and discuss about who is going update what.

**Review by lowkaiwei98 — APPROVED** [review]
lgtm

**Pull request merged**